arbor/backends/multicore/matrix_state.hpp

```diff
@@ -116,7 +118,7 @@ struct matrix_state {
         for (auto cv_span: util::partition_view(cell_cv_divs)) {
             auto first = cv_span.first;
             auto last = cv_span.second; // one past the end
-
+            if (first >= last) continue; // skip empty / reversed partitions. Bugs lurk.
```

Comment:
Instead of the comment referring to the
```C++
if (first >= last) continue; // skip cell with no CVs
```

Reply:
Sure.

Reply:
Question: @halfflat mentioned that empty partitions might arise due to other reasons. This check incidentally covers the 'no CV' case, but will also skip those other cases. Therefore, this comment might be misleading.